tests/issues.rs

```diff
@@ -0,0 +1,10 @@
+use clap::{App, Arg};
+
+#[test]
+fn issue_2022() {
```

Comment:
Can you please put this in `tests/help.rs`?

Reply:
`tests/parser.rs` / `tests/opts.rs` maybe? This issue has nothing to do with help

Reply:
opts.rs please